### ISSUE TITLE
How to get value out?

### ISSUE THREAD

**Issue by deyceg** [apertureless/vue-password-strength-meter#1]
### Expected Behavior

input events emitted to parent

### Actual Behavior

No event emitted so unable to observe password value

From Vue.js Docs:

> So for a component to work with v-model, it must:
> accept a value prop
> emit an input event with the new value

### Environment
- OS: Mac
- NPM version: 3.x
- vue.js version: 2.x
- vue-password-strength-meter version: 0.1.2

**Comment by apertureless** [comment]
Thanks for your feedback. Yeah you're right. Right now it does not emit an event.
Wasn't nessasary for my specific usecase.

But it's a good point to make it more useable and flexible. 

Will be implemented.  💪

**Comment by apertureless** [comment]
It's implemented in `v0.2.0` as it's an api break.

## Usage would be now:

```javascript
<script>
  import Password from 'vue-password-strength-meter'
  export default {
    components: { Password }
    data () {
      return {
        password: null
      }
    }
  }
</script>

<template>
  <password v-model="password"></password>
</template>
```
